mix.exs

```diff
@@ -2,7 +2,7 @@ defmodule Telepoison.MixProject do
   use Mix.Project
 
   @source_url "https://github.com/primait/telepoison"
-  @version "1.1.2"
+  @version "1.2.0"
```

Comment:
I don't think we should bump the package version as a part of this PR?
Also this seems like a minor change, since I don't think it's even observable from the end-user perspective

Reply:
I bumped it because of a previous attempt to release but it doesn't make sense, I reverted the bump